manifests/server/conf.pp

```diff
@@ -131,7 +131,7 @@
   Boolean                          $enable_fallback_ciphers         = true,
   Variant[Boolean,Enum['delayed']] $compression                     = false,
   Ssh::Syslogfacility              $syslogfacility                  = 'AUTHPRIV',
-  Boolean                          $gssapiauthentication            = false,
+  Boolean                          $gssapiauthentication            = $::ssh::server::params::gssapiauthentication,
```

Comment:
_If_ the SSH client should default to the same GSSAPI configuration as the SSH server, then this should refer to a new parameter in `init.pp` (which defaults to `$::ssh::server::params::gssapiauthentication`)

Reply:
I decided against putting the logic in init.pp to fit in with the architecture of the module as it exists today.